app/views/layouts/application.html.haml

```diff
@@ -52,6 +52,11 @@
                 - current_site.posts.published.order_by_recently.limit(5).each do |post|
                   = render partial: 'posts/summary_with_content', locals: {post: post}
 
+            - unless Rails.env.production?
```

Comment:
Why you ignore this code on production?
I guess you want to merge this without design, right?